dotnet/src/IntegrationTests/TemplateLanguage/tests.txt

```diff
@@ -18,9 +18,6 @@
 {{}}
 {{}}
 
-.{{asis}}.
-..
-
```

Comment:
Why is this change required?

Reply:
The "asis" function has a required string input:
```
        [SKFunction, Description("This is a test"), SKName("asis")]
        public string MyFunction2(string input)
        {
            return input;
        }
```

Calling `asis` on its own is will crash because there's no input value provided. I can update asis to accept a nullable `string? input = null`

Reply:
undone